deployment/fedora-package-x64/pkg-src/jellyfin.env

```diff
@@ -14,15 +14,22 @@
 # General options
 #
 
-# Tell jellyfin wich ffmpeg/ffprobe to use
-# JELLYFIN_FFMPEG="--ffmpeg /usr/bin/ffmpeg --ffprobe /usr/bin/ffprobe"
-
 # Program directories
 JELLYFIN_DATA_DIRECTORY="/var/lib/jellyfin"
 JELLYFIN_CONFIG_DIRECTORY="/etc/jellyfin"
 JELLYFIN_LOG_DIRECTORY="/var/log/jellyfin"
 JELLYFIN_CACHE_DIRECTORY="/var/cache/jellyfin"
+
 # In-App service control
-JELLYFIN_RESTART_OPT="--restartpath /usr/libexec/jellyfin/restart.sh"
-# Additional options for the binary
-JELLYFIN_ADD_OPTS=""
+JELLYFIN_RESTART_OPT="--restartpath=/usr/libexec/jellyfin/restart.sh"
+
+# [OPTIONAL] ffmpeg binary paths, overriding the UI-configured values
+#JELLYFIN_FFMPEG_OPT="--ffmpeg=/usr/bin/ffmpeg"
+#JELLYFIN_FFPROBE_OPT="--ffprobe=/usr/bin/ffprobe"
+
+# [OPTIONAL] run Jellyfin as a headless service
+#JELLYFIN_SERVICE_OPT="--service"
```

Comment:
What does the `--service` option do?
I cant make any sense out `run as headless service` does it mean fork and daemonize?

Reply:
Yup, I have no idea if these will ever be useful, but because of how I'm doing args it makes sense to have this in here, commented by default, in case anyone wants it, so they don't have to mess with the unit file.